packages/docs-reanimated/docs/guides/web-support.mdx

```diff
@@ -4,33 +4,56 @@ title: Web Support
 sidebar_label: Web Support
 ---
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
-
-<DocsCompatibilityInfo />
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 It's possible to launch Reanimated in a web browser. For that case all of the functionalities are implemented purely in JavaScript, hence the efficiency of the animations might be lower.
 
-Reanimated for Web requires the following configuration steps. You need to add [`@babel/plugin-proposal-export-namespace-from`](https://babeljs.io/docs/en/babel-plugin-proposal-export-namespace-from) as well as Reanimated Babel plugin to your `babel.config.js`.
+### Step 1: Install the package
 
-```bash
-yarn add @babel/plugin-proposal-export-namespace-from
-```
+Install [`@babel/plugin-proposal-export-namespace-from`](https://babeljs.io/docs/babel-plugin-transform-export-namespace-from):
 
-```js {5,6}
-module.exports = {
-  presets: [
-      ...
-  ],
-  plugins: [
-    ...
-    '@babel/plugin-proposal-export-namespace-from',
-    'react-native-reanimated/plugin',
-  ],
-};
+<Tabs groupId="package-managers">
+  <TabItem value="expo" label="EXPO" default>
+
+    npx expo install @babel/plugin-proposal-export-namespace-from
+
+  </TabItem>
+  <TabItem value="npm" label="NPM">
+
+    npm install @babel/plugin-proposal-export-namespace-from
+
+  </TabItem>
+  <TabItem value="yarn" label="YARN">
+
+    yarn add @babel/plugin-proposal-export-namespace-from
+
+  </TabItem>
+</Tabs>
+
+### Step 2: Add plugins to config file
+
+Add `@babel/plugin-proposal-export-namespace-from` and Reanimated Babel plugin to your `babel.config.js`.
+
+```js {7,8}
+  module.exports = {
+      presets: [
+        ... // don't add it here :)
+      ],
+      plugins: [
+          ...
+          '@babel/plugin-proposal-export-namespace-from',
+          'react-native-reanimated/plugin',
+      ],
+  };
 ```
 
+Make sure to list `react-native-reanimated/plugin` last.
```

Comment:
I'd make it stand out because it is important, such as here:
https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/#step-2-add-reanimateds-babel-plugin

Reply:
Maybe `caution` block 😄